src/main/java/tds/common/data/mysql/spring/TdsPropertyRowMapper.java

```diff
@@ -2,19 +2,24 @@
 
 import org.springframework.beans.BeanWrapper;
 import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import tds.common.data.mysql.InstantPropertyEditorSupport;
 import tds.common.data.mysql.UuidPropertyEditorSupport;
 
+import java.time.Instant;
 import java.util.UUID;
 
+/**
+ * Handle MySQL-specific conversions of data stored in the TDS databases.
+ */
+public class TdsPropertyRowMapper<T> extends BeanPropertyRowMapper<T> {
 
-public class UuidBeanPropertyRowMapper<T> extends BeanPropertyRowMapper<T> {
-
-    public UuidBeanPropertyRowMapper(Class<T> type) {
+    public TdsPropertyRowMapper(Class<T> type) {
```

Comment:
My thinking here is that will offer projects more flexibility in configure their property mappers.